build-parent/pom.xml

```diff
@@ -33,7 +33,7 @@
         <!-- These properties are needed in order for them to be resolvable by the documentation -->
         <!-- The Graal version we suggest using in documentation - as that's
            what we work with by self downloading it: -->
-        <graal-sdk.version-for-documentation>20.1.0</graal-sdk.version-for-documentation>
+        <graal-sdk.version-for-documentation>20.2.0</graal-sdk.version-for-documentation>
         <mandrel.version-for-documentation>20.1.0.1.Final</mandrel.version-for-documentation>
```

Comment:
@galderz is there a 20.2.0-compatible Mandrel version to be released soon?

Reply:
We're working to release a Mandrel 20.2.0 beta. There's one PR we'd like to get it to fix an timezone name issue. For 20.2.0 final, we need to wait until JDK 11.0.9 comes out.